nodejs/week2/mealSharing/src/bakckend/routes/meals.js

```diff
@@ -0,0 +1,83 @@
+const express = require("express");
+const router = express.Router();
+const meals = require("../data/reviewed-meals");
+
+router.get("/", (req, res) => {
+	const maxPrice = req.query.maxPrice;
+	const title = req.query.title;
+	const createdAfter = req.query.createdAfter;
+	const limit = req.query.limit;
+
+	if (maxPrice) {
+		const maxPriceValue = parseInt(maxPrice);
+		if (!maxPriceValue) {
+			res.status(404).send(`Enter price in number`);
```

Comment:
not sending a maxPriceValue is OK, this is an optional query parameter. If it is not sent with the request you just don't use it to filter the meals.